src/pages/MetricDetail.svelte

```diff
@@ -343,6 +343,19 @@
               >
               <CopyButton textToCopy={metric.event_info.name} />)
             {/if}
+            {#if metric.variants.length}
+              <p>
+                (also in <AuthenticatedLink
+                  href={`https://mozilla.acryl.io/dataset/urn:li:dataset:(urn:li:dataPlatform:bigquery,moz-fx-data-shared-prod.${params.app}.${selectedPingVariant.id},PROD)`}
```

Comment:
@Iinh should this actually be more explicit and have its own entry as "Data Catalogue" (e.g. below Big Query)? If you think that doesn't help, what about listing the tables as bullets instead of having this long text?

For example

```markdown
* `org_mozilla_ios_firefox.events` (bla bla bla)
* `firefox_ios.events` (bla bla bla)
```

My concern is that the discovery of this information is hindered by the density :)

Reply:
Since this is for the Metric page, and the Data Catalog link is of a ping dataset, I don't think it'd make sense to have its own separate entry yet (we're reserving that spot for when Glean metrics become available in Data Catalog 🙂)

f you think that doesn't help, what about listing the tables as bullets instead of having this long text?

Reply:
> > If you think that doesn't help, what about listing the tables as bullets instead of having this long text?
> 
> Makes sense! I'll change it :)

Thanks!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Reply:
I simplified it to just one line, instead of using bullet points:

<img width="844" alt="CleanShot 2023-05-23 at 21 55 04@2x" src="https://github.com/mozilla/glean-dictionary/assets/28797553/4741cd7b-70e0-447b-8395-17bc5db0b6fa">

What do you think?